core/src/main/scala/org/apache/spark/ui/env/EnvironmentPage.scala

```diff
@@ -25,12 +25,20 @@ import org.apache.spark.ui.{UIUtils, WebUIPage}
 
 private[ui] class EnvironmentPage(parent: EnvironmentTab) extends WebUIPage("") {
   private val listener = parent.listener
-
+  val passwordProperties: Seq[String] = Seq("spark.ssl.keyPassword",
+    "spark.ssl.keyStorePassword", "spark.ssl.trustStorePassword")
+
+  def removePass(kv: (String, String)): (String, String) = {
+    if (passwordProperties.contains(kv._1)) {
```

Comment:
Based on @srowen suggestion in the JIRA, maybe we should do a substring check for any prop with "password" in it, that could make this more of a generic check. Or is there a prop with "password" in it that we wouldn't want to obfuscate?